pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -867,4 +865,12 @@ spec:
       name: {{ .Volume.owner }}
 ---
 `
+
+// cstorVolumeYamlsFor070 returns all the yamls related to cstor volume in a
+// string format
+//
+// NOTE:
+//  This is an implementation of MultiYamlFetcher
+func cstorVolumeYamlsFor070() string {
```

Comment:
We need not have a function if we settle on a constant.

Reply:
yes I agree, for that, I think we should have one more function similar to `ParseArtifactListFromMultipleYamls for parsing string constants.